mk/test.mk

```diff
@@ -22,10 +22,10 @@ GINKGO_TEST:=$(GINKGO) $(GOFLAGS) $(LD_FLAGS) $(GINKGO_TEST_FLAGS)
 
 .PHONY: test
 test:
```

Comment:
Is it just simpler to add `build/epbf` here? IMO it's not that much *work* and it's less error prone to do it always instead of adding it to each test target

Reply:
ack

Reply:
addressed